sass/form/_index.scss

```diff
@@ -1,23 +1,33 @@
 form {
   // setting the font size here sizes everything within
-  margin-bottom: var(--spacing-s);
+  &:not(:last-child) {
+    margin-bottom: var(--spacing-s);
+  }
 
-  button,
+  button:not(.button--link),
   input,
   select {
     height: var(--spacing-l);
     border: 1px solid;
   }
 
+  select,
+  radio-element,
+  checkbox-element {
+    cursor: pointer;
+  }
+
   [type="email"],
-  [type="text"] {
+  [type="text"],
+  [type="number"],
+  [type="password"] {
     padding-right: var(--spacing-s);
     padding-left: var(--spacing-s);
     transition: border 0.2s;
   }
 
-  [type="button"],
-  [type="submit"] {
+  [type="submit"],
+  .button--primary {
```

Comment:
So we can have links/inverse buttons in the form for minor stuff.

Reply:
👍🏾